### PR TITLE
Kp todo

### LINKED ISSUE #16: KP_TODO
Spots that @kaizadp needs to look at are marked with `KP_TODO`.

### PR DESCRIPTION
**Pull request by kaizadp**
- fixed KP_TODO:
  - function descriptions
  - Hertkorn2013 data description/rda
  - @bpbond Does this look ok?
- rename `gg_spectra` -> `nor_plot_spectra`
- updated function descriptions (.Rd files)
- added authors

Closes #16

### CODE REVIEW

File: R/1-functions_graphs.R [hunk 1]
```diff
@@ -14,14 +14,14 @@
...
 #' @param STAGGER how much to stagger the labels?
 #'
-#' @return A dataframe with columns describing ... KP_TODO
+#' @return A ggplot object
 #'
 #' @import ggplot2
 #' @importFrom dplyr %>% distinct filter mutate select left_join rename
 #' @importFrom tidyr drop_na replace_na
 #' @importFrom utils head
 #' @export
```
Comment (bpbond) [inline R/1-functions_graphs.R:23]: Provide an `@examples`?

File: R/1-functions_processing.R [hunk 2]
```diff
@@ -4,7 +4,7 @@
 #' @description Imports multiple spectra files and then combines and cleans the data.
 #'
 #' @param path Directory where the spectra files are saved
-#' @param method KP_TODO
+#' @param method software used for initial processing of NMR spectra (before using this package). Available options include "mnova" and "topspin".
```
Comment (bpbond) [inline R/1-functions_processing.R]: "Software"

File: R/data.R [hunk 3]
```diff
@@ -1,14 +1,20 @@
...
 #' NMR grouping bins from Clemente et al. (2012).
 #'
-#' NMR grouping bins from Clemente et al. (2012). KP_TODO
+#' NMR grouping bins from Clemente et al. (2012), using DMSO-D6 as solvent.
+#' (1) aliphatic polymethylene and methyl groups (0.6–1.3 ppm, “aliphatic1”);
+#' (2) aliphatic methyl and methylene near O and N (1.3–2.9 ppm, “aliphatic2”);
+#' (3) O-alkyl, mainly from carbohydrates and lignin (2.9–4.1 ppm);
+#' (4) α-proton of peptides (4.1–4.8 ppm);
+#' (5) aromatic and phenolic (6.2–7.8 ppm); and
+#' (6) amide, from proteins (7.8–8.4 ppm).
```
Comment (bpbond) [inline R/data.R:10]: Beautiful—thank you!

**Review by bpbond — CHANGES_REQUESTED** [review]
Looks good! A few minor suggestions below.

**Comment by kaizadp** [comment]
Thanks, changes made!

fbd85503a1c312d083f5a3f4ed88c2acbf8b0c9a 
237a05761c66273cb219503b6b950a67e99e3f35

The `nmr_plot_spectra` example is a little weird for meb_burn data, because the intensities are very high and have not been normalized. Will update the example later, when Morgan gives me simpler, normalized data.

**Comment by bpbond** [comment]
So as a general rule of thumb: please don't merge PRs with failing tests. I will look at this later.

**Comment by kaizadp** [comment]
Good point! I think I know what’s causing the tests to fail, so I’m working on it right now.
Next time, I’ll fix before merging. 
Thanks!

**Comment by bpbond** [comment]
Thanks! Sorry if that sounded grumpy :)

**Comment by kaizadp** [comment]
No worries, I was impatient to merge branches.

PS: there's still an error. The previous error was because I changed the function name (`nmr_plot_spectra`) but didn't update the tests and documentation. I did so, and now I get this error:
(not urgent, when you get a chance later)

image

**Pull request merged**